docs/News.md

```diff
@@ -1,3 +1,11 @@
+## May 4, 2020
+*   **Added data** consisting of 574 samples from 3 studies:
+     * [Uterine Sarcoma/Mesenchymal (MSK, Clin Cancer Res 2020)](https://www.cbioportal.org/study/summary?id=usarc_msk_2020) *108 samples* 
+     * [Metastatic castration-sensitive prostate cancer (MSK, Clin Cancer Res 2020)](https://www.cbioportal.org/study/summary?id=prad_mcspc_mskcc_2020) *424 samples* 
+    *  [Glioblastoma (Columbia, Nat Med. 2019)] (https://www.cbioportal.org/study/summary?id=gbm_columbia_2019) *42 samples* 
```

Comment:
I couldn't find the gbm study...

Reply:
@jj The gbm study is being imported to public overnight, will be there tomorrow morning.

Reply:
Maybe change the release date to May 5 then?

Reply:
@jjgao changed to May 6. Re-importing the study again with the OS PD1i change tonight.